frontend/lib/identification/user_code_model.dart

```diff
@@ -1,15 +1,16 @@
 import 'dart:developer';
 
+import 'package:ehrenamtskarte/build_config/build_config.dart';
 import 'package:ehrenamtskarte/identification/user_code_store.dart';
 import 'package:ehrenamtskarte/proto/card.pb.dart';
 import 'package:flutter/foundation.dart';
 
 class UserCodeModel extends ChangeNotifier {
```

Comment:
```suggestion
class UserCodesModel extends ChangeNotifier {
```

Reply:
hm not sure if we should refactor this and all occurrences